postgres/README.md

```diff
@@ -87,26 +87,26 @@ To configure this check for an Agent running on a host:
       #
       - host: localhost
 
-        ## @param port - integer - required
-        ## Port to use when connecting to PostgreSQL.
+        ## @param port - integer - optional - default: 5432
+        ## The port to use when connecting to PostgreSQL.
         #
-        port: 5432
-
-        ## @param user - string - required
-        ## Datadog Username created to connect to PostgreSQL.
+        # port: 5432
+    
+        ## @param username - string - required
+        ## The Datadog username created to connect to PostgreSQL.
         #
         username: datadog
-
-        ## @param pass - string - required
-        ## Password associated with the Datadog user.
+    
+        ## @param password - string - optional
+        ## The password associated with the Datadog user.
         #
-        password: "<PASSWORD>"
-
+        # password: <PASSWORD>
+    
         ## @param dbname - string - optional - default: postgres
-        ## Name of the PostgresSQL database to monitor.
-        ## Note: If omitted, the default system postgres database is queried.
+        ## The name of the PostgresSQL database to monitor.
+        ## Note: If omitted, the default system Postgres database is queried.
         #
-        dbname: "<DB_NAME>"
+        # dbname: <DBNAME>
```

Comment:
Same as above, though since this is the second instance, I assume that commenting out the line is intentional.

Reply:
same 
yes, this was in the old examples: https://github.com/DataDog/integrations-core/blob/7.19.x/postgres/datadog_checks/postgres/data/conf.yaml.example. (v.7.19) - now is optional.